.github/workflows/unit.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: unit
+
+"on":
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-2019, windows-2016]
+        # Due to https://github.com/actions/runner/issues/849, we have to use quotes for '3.0'
+        ruby: [2.7, '3.0']
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+        bundler-cache: true
+    - run: bundle exec rake spec:unit
+    - run: bundle exec rake component_specs
```

Comment:
why unit tests? That seems like not what the PR says?

Reply:
consider this a nice WIP to see how far I can take actions w/o test refactoring. Turns out unit tests on Windows are going to require refactoring since they assume C: and the tests run on D:

Reply:
Heh. Of course they do. Because windows. :)